src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderTestData.java

```diff
@@ -201,14 +201,16 @@ public static SnapshotBuilderCohort createCohort() {
         .name("cohort")
         .addCriteriaGroupsItem(
             new SnapshotBuilderCriteriaGroup()
+                .meetAll(true)
+                .mustMeet(true)
                 .addCriteriaItem(
                     new SnapshotBuilderProgramDataListCriteria()
                         .id(0)
                         .kind(SnapshotBuilderCriteria.KindEnum.LIST))
                 .addCriteriaItem(
                     new SnapshotBuilderDomainCriteria()
                         .conceptId(100)
-                        .id(19)
+                        .id(CONDITION_OCCURRENCE_DOMAIN_ID)
```

Comment:
use the same id as used in the test data settings so criteriaQueryBuilder test passes